.travis.yml

```diff
@@ -61,10 +61,11 @@ deploy:
   - pmd-dist/target/pmd-*.zip
   - docs/pmd-doc-*.zip
   skip_cleanup: true
+  draft: true
   on:
     tags: true
     repo: pmd/pmd
-    condition: "${TRAVIS_SECURE_ENV_VARS} = true && ${BUILD} != sonar && ${BUILD} != coveralls"
+    condition: "${TRAVIS_SECURE_ENV_VARS} = true && ${BUILD} != sonar && ${BUILD} != coveralls && ${TRAVIS_OS_NAME} != osx"
```

Comment:
Since windows will need to be added to the condition here, I guess, it's easier to move this check into `.travis/release.sh` and just exit there, if we are not on linux and not doing "deploy" or "docs".

Reply:
Fixed via f10b631d76c34908ea1adcb41bcf1d71a184e0ba

The check can't be moved into the `release.sh` script, since travis will then execute the deployment provider "github releases" anyway - which it should only do for the linux build and the doc build.